scan

```diff
@@ -288,17 +313,17 @@ def scan_domains(scanners, domains, options):
     logging.warn("Results written to CSV.")
 
     # Save metadata.
-    end_time = utils.local_now()
+    end_time = scan_utils.local_now()
     duration = end_time - start_time
     metadata = {
-        'start_time': utils.utc_timestamp(start_time),
-        'end_time': utils.utc_timestamp(end_time),
-        'duration': utils.just_microseconds(duration),
+        'start_time': scan_utils.utc_timestamp(start_time),
+        'end_time': scan_utils.utc_timestamp(end_time),
+        'duration': scan_utils.just_microseconds(duration),
         'durations': durations,
         'command': start_command,
         'scan_uuid': scan_uuid
```

Comment:
Assigning this got moved lower in the script. Is this still set by this time?

Reply:
Do you mean the variables `durations`, `start_command`, and `scan_uuid`? If so, yes, they're still in scope at this point. (`start_command` is still a global variable.)